modules/bitgo/example/js/ada/create-wallet.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Create a TSS Cardano wallet at BitGo.
+ * This makes use of the convenience function generateWallet
+ *
+ * This tool will help you see how to use the BitGo API to easily create a wallet.
+ * In this form, it creates 2 keys on the host which runs this example.
+ * It is HIGHLY RECOMMENDED that you GENERATE THE KEYS ON SEPARATE MACHINES for real money wallets!
+ *
+ * To perform more advanced features, such as encrypting keys yourself, please look at createWalletAdvanced.js
+ *
+ * Copyright 2022, BitGo, Inc.  All Rights Reserved.
+ */
+const BitGo = require('bitgo');
+const bitgo = new BitGo.BitGo({
+  env: 'custom',
+  customRootURI: 'https://testnet-07-app.bitgo-dev.com',
```

Comment:
I'd suggest to leave this empty (especially because this is a public repo) and let the user set it

Reply:
Ah, I did not catch this part. Let's please use external facing hosts